src/components/options/useCommentOptions.ts

```diff
@@ -107,6 +110,9 @@ export const useCommentOptions = ({
                 },
               };
             }
+            case CommentOptions.DEBUG: {
+              return useDebugOptions({ comment, commentChain });
```

Comment:
Rather than pass both, you can simply pass `commentChain` and then check if `commentChain` has a single element in it, in which case you can treat it as a single comment.

If I'm getting this wrong let me know. The code might need to be rewritten to make this clearer anyway.